Documentation/Helm-Charts/operator-chart.md

```diff
@@ -146,6 +146,7 @@ The following table lists the configurable parameters of the rook-operator chart
 | `rbacEnable` | If true, create & use RBAC resources | `true` |
 | `resources` | Pod resource requests & limits | `{"limits":{"cpu":"500m","memory":"512Mi"},"requests":{"cpu":"100m","memory":"128Mi"}}` |
 | `scaleDownOperator` | If true, scale down the rook operator. This is useful for administrative actions where the rook operator must be scaled down, while using gitops style tooling to deploy your helm charts. | `false` |
+| `serviceMonitor` | Service monitor for Ceph CSI driver | `{"enabled":false,"interval":"5s"}` |
```

Comment:
Let's have separate settings for enabled and interval, instead of combining them.

Reply:
The docs need to be updated in operator-chart.gotmpl.md, then this file (operator-chart.md) is generated.